app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -18,7 +18,7 @@ class Admin::AdminSetsController < ApplicationController
     self.member_search_builder_class = Hyrax::AdminAdminSetMemberSearchBuilder
 
     # Used to get a list of admin sets for the index action
-    self.list_search_builder_class = Hyrax::AdminSetSearchBuilder
+    # self.list_search_builder_class = Hyrax::AdminSetSearchBuilder
```

Comment:
Determined this wasn't being used for the index function, line 36 in the index function is actually getting the admin sets.  Since scoping via edit works for both admins and users with manage access, commented code removed.